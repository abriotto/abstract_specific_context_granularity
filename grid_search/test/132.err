  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s]  1%|#8                                                                                                                                                                                          | 1/99 [00:00<00:13,  7.31it/s]  2%|###7                                                                                                                                                                                        | 2/99 [00:00<00:12,  7.57it/s]  5%|#########4                                                                                                                                                                                  | 5/99 [00:00<00:08, 11.11it/s]  8%|###############1                                                                                                                                                                            | 8/99 [00:00<00:07, 12.34it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:07, 11.38it/s] 13%|########################5                                                                                                                                                                  | 13/99 [00:01<00:06, 12.89it/s] 15%|############################3                                                                                                                                                              | 15/99 [00:01<00:06, 12.34it/s] 17%|################################1                                                                                                                                                          | 17/99 [00:01<00:06, 13.19it/s] 19%|###################################8                                                                                                                                                       | 19/99 [00:01<00:06, 12.31it/s] 21%|#######################################6                                                                                                                                                   | 21/99 [00:01<00:06, 11.79it/s] 23%|###########################################4                                                                                                                                               | 23/99 [00:01<00:06, 11.42it/s] 25%|###############################################2                                                                                                                                           | 25/99 [00:02<00:07,  9.70it/s] 27%|##################################################9                                                                                                                                        | 27/99 [00:02<00:06, 10.94it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:02<00:07,  9.34it/s] 31%|##########################################################5                                                                                                                                | 31/99 [00:02<00:07,  9.61it/s] 33%|##############################################################3                                                                                                                            | 33/99 [00:02<00:06, 10.63it/s] 35%|##################################################################1                                                                                                                        | 35/99 [00:03<00:06,  9.75it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:03<00:07,  8.30it/s] 38%|#######################################################################7                                                                                                                   | 38/99 [00:03<00:07,  7.84it/s] 40%|###########################################################################5                                                                                                               | 40/99 [00:03<00:07,  7.96it/s] 42%|###############################################################################3                                                                                                           | 42/99 [00:04<00:07,  8.11it/s] 43%|#################################################################################2                                                                                                         | 43/99 [00:04<00:07,  7.43it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:04<00:07,  7.61it/s] 46%|######################################################################################8                                                                                                    | 46/99 [00:04<00:07,  7.29it/s] 48%|##########################################################################################6                                                                                                | 48/99 [00:04<00:05,  9.37it/s] 51%|##############################################################################################4                                                                                            | 50/99 [00:05<00:06,  7.56it/s] 53%|##################################################################################################2                                                                                        | 52/99 [00:05<00:06,  7.72it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:05<00:06,  7.44it/s] 55%|#####################################################################################################9                                                                                     | 54/99 [00:05<00:06,  7.16it/s] 56%|#######################################################################################################8                                                                                   | 55/99 [00:06<00:06,  6.81it/s] 58%|###########################################################################################################6                                                                               | 57/99 [00:06<00:05,  7.76it/s] 59%|#############################################################################################################5                                                                             | 58/99 [00:06<00:05,  7.03it/s] 60%|###############################################################################################################4                                                                           | 59/99 [00:06<00:05,  6.73it/s] 62%|###################################################################################################################2                                                                       | 61/99 [00:06<00:04,  8.39it/s] 63%|#####################################################################################################################1                                                                     | 62/99 [00:06<00:04,  7.83it/s] 65%|########################################################################################################################8                                                                  | 64/99 [00:07<00:03,  9.20it/s] 66%|##########################################################################################################################7                                                                | 65/99 [00:07<00:04,  8.29it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:07<00:03,  8.27it/s] 70%|##################################################################################################################################3                                                        | 69/99 [00:07<00:02, 10.20it/s] 72%|######################################################################################################################################1                                                    | 71/99 [00:07<00:02, 10.82it/s] 74%|#########################################################################################################################################8                                                 | 73/99 [00:08<00:02,  8.72it/s] 75%|###########################################################################################################################################7                                               | 74/99 [00:08<00:03,  8.09it/s] 76%|#############################################################################################################################################6                                             | 75/99 [00:08<00:03,  7.63it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:08<00:03,  7.10it/s] 78%|#################################################################################################################################################4                                         | 77/99 [00:08<00:03,  6.78it/s] 80%|#####################################################################################################################################################2                                     | 79/99 [00:08<00:02,  7.57it/s] 81%|#######################################################################################################################################################1                                   | 80/99 [00:09<00:02,  7.01it/s] 82%|#########################################################################################################################################################                                  | 81/99 [00:09<00:02,  6.59it/s] 84%|############################################################################################################################################################7                              | 83/99 [00:09<00:02,  7.25it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:09<00:02,  6.86it/s] 87%|##################################################################################################################################################################4                        | 86/99 [00:09<00:01,  8.96it/s] 88%|####################################################################################################################################################################3                      | 87/99 [00:09<00:01,  8.23it/s] 90%|########################################################################################################################################################################1                  | 89/99 [00:10<00:01,  8.25it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:10<00:00,  8.43it/s] 94%|###############################################################################################################################################################################6           | 93/99 [00:10<00:00,  9.51it/s] 96%|###################################################################################################################################################################################4       | 95/99 [00:10<00:00,  9.18it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:11<00:00, 10.37it/s]100%|###########################################################################################################################################################################################| 99/99 [00:11<00:00,  8.89it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s]  4%|#######5                                                                                                                                                                                    | 1/25 [00:00<00:04,  5.82it/s] 12%|######################5                                                                                                                                                                     | 3/25 [00:00<00:02,  7.59it/s] 20%|#####################################6                                                                                                                                                      | 5/25 [00:00<00:02,  7.91it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:01,  9.61it/s] 36%|###################################################################6                                                                                                                        | 9/25 [00:01<00:02,  7.70it/s] 44%|##################################################################################2                                                                                                        | 11/25 [00:01<00:01,  7.96it/s] 48%|#########################################################################################7                                                                                                 | 12/25 [00:01<00:01,  7.46it/s] 56%|########################################################################################################7                                                                                  | 14/25 [00:01<00:01,  9.31it/s] 64%|#######################################################################################################################6                                                                   | 16/25 [00:01<00:00, 10.44it/s] 72%|######################################################################################################################################6                                                    | 18/25 [00:02<00:00,  9.67it/s] 80%|#####################################################################################################################################################6                                     | 20/25 [00:02<00:00,  9.33it/s] 84%|#############################################################################################################################################################                              | 21/25 [00:02<00:00,  8.54it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:02<00:00,  7.79it/s] 96%|###################################################################################################################################################################################5       | 24/25 [00:02<00:00,  7.88it/s]100%|###########################################################################################################################################################################################| 25/25 [00:02<00:00,  8.50it/s]
