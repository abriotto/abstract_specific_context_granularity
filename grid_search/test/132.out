# ["--attributes=3", "--values=4", "--game_size=10", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=128", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.97", "--temperature=1"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6962377429008484, "acc": 0.5055881142616272, "length": 3.4806020259857178, "mode": "train", "epoch": 1}
{"loss": 0.6947744488716125, "acc": 0.4862876534461975, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6866150498390198, "acc": 0.5495394468307495, "length": 2.695343255996704, "mode": "train", "epoch": 2}
{"loss": 0.6716985702514648, "acc": 0.6504099369049072, "length": 2.177257537841797, "mode": "test", "epoch": 2}
{"loss": 0.6469141244888306, "acc": 0.6790063977241516, "length": 3.1810221672058105, "mode": "train", "epoch": 3}
{"loss": 0.6040696501731873, "acc": 0.6919780969619751, "length": 3.7826087474823, "mode": "test", "epoch": 3}
{"loss": 0.5788534879684448, "acc": 0.6983393430709839, "length": 3.6236116886138916, "mode": "train", "epoch": 4}
{"loss": 0.5692497491836548, "acc": 0.7288896441459656, "length": 3.4732441902160645, "mode": "test", "epoch": 4}
{"loss": 0.5199813842773438, "acc": 0.7535826563835144, "length": 3.5974371433258057, "mode": "train", "epoch": 5}
{"loss": 0.5138632655143738, "acc": 0.7653579711914062, "length": 3.9698996543884277, "mode": "test", "epoch": 5}
{"loss": 0.4595170021057129, "acc": 0.8020252585411072, "length": 3.9330973625183105, "mode": "train", "epoch": 6}
{"loss": 0.46716567873954773, "acc": 0.7903009653091431, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.4124067425727844, "acc": 0.8255982398986816, "length": 3.9861953258514404, "mode": "train", "epoch": 7}
{"loss": 0.4486282765865326, "acc": 0.8037624955177307, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.3897615671157837, "acc": 0.8344831466674805, "length": 3.9931395053863525, "mode": "train", "epoch": 8}
{"loss": 0.4277219772338867, "acc": 0.813879668712616, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.37051135301589966, "acc": 0.8460143208503723, "length": 3.9958877563476562, "mode": "train", "epoch": 9}
{"loss": 0.42136359214782715, "acc": 0.8199832439422607, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.3599628806114197, "acc": 0.8497340083122253, "length": 3.9948220252990723, "mode": "train", "epoch": 10}
{"loss": 0.3943069577217102, "acc": 0.832023561000824, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.3420160114765167, "acc": 0.8602932691574097, "length": 3.9981331825256348, "mode": "train", "epoch": 11}
{"loss": 0.3766739070415497, "acc": 0.8453176021575928, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.3310089707374573, "acc": 0.8649248480796814, "length": 3.998328685760498, "mode": "train", "epoch": 12}
{"loss": 0.3673517405986786, "acc": 0.8489965796470642, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.3271523416042328, "acc": 0.8674653172492981, "length": 3.9980900287628174, "mode": "train", "epoch": 13}
{"loss": 0.363038033246994, "acc": 0.8541806936264038, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.318758487701416, "acc": 0.871813178062439, "length": 3.997325897216797, "mode": "train", "epoch": 14}
{"loss": 0.3519543409347534, "acc": 0.8578596115112305, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.30952179431915283, "acc": 0.8749264478683472, "length": 3.994925022125244, "mode": "train", "epoch": 15}
{"loss": 0.3418533504009247, "acc": 0.8617055416107178, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.30337682366371155, "acc": 0.8775538206100464, "length": 3.998647451400757, "mode": "train", "epoch": 16}
{"loss": 0.33009666204452515, "acc": 0.8699833750724792, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.29862815141677856, "acc": 0.8802544474601746, "length": 3.9980926513671875, "mode": "train", "epoch": 17}
{"loss": 0.3322109878063202, "acc": 0.8713211417198181, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.29234081506729126, "acc": 0.8845400810241699, "length": 3.999284505844116, "mode": "train", "epoch": 18}
{"loss": 0.3081245720386505, "acc": 0.8791806101799011, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.2813186049461365, "acc": 0.8888630867004395, "length": 3.999641180038452, "mode": "train", "epoch": 19}
{"loss": 0.314666748046875, "acc": 0.8784281611442566, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.2778739631175995, "acc": 0.8915631175041199, "length": 3.999159336090088, "mode": "train", "epoch": 20}
{"loss": 0.3019307255744934, "acc": 0.881939709186554, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.2736104130744934, "acc": 0.8927099108695984, "length": 3.99880051612854, "mode": "train", "epoch": 21}
{"loss": 0.30050843954086304, "acc": 0.8782609701156616, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.2703852653503418, "acc": 0.8911280632019043, "length": 3.9983699321746826, "mode": "train", "epoch": 22}
{"loss": 0.3070502281188965, "acc": 0.8816889524459839, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.2631359100341797, "acc": 0.8979039192199707, "length": 3.999584436416626, "mode": "train", "epoch": 23}
{"loss": 0.29756781458854675, "acc": 0.8837790489196777, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.26237937808036804, "acc": 0.8966104388237, "length": 3.9986419677734375, "mode": "train", "epoch": 24}
{"loss": 0.2919251024723053, "acc": 0.8818561434745789, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.25555065274238586, "acc": 0.897929847240448, "length": 3.999290943145752, "mode": "train", "epoch": 25}
{"loss": 0.286961168050766, "acc": 0.8871236443519592, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.25351834297180176, "acc": 0.900323748588562, "length": 3.999258041381836, "mode": "train", "epoch": 26}
{"loss": 0.2797108590602875, "acc": 0.8911371827125549, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.24416059255599976, "acc": 0.9063318371772766, "length": 3.9998855590820312, "mode": "train", "epoch": 27}
{"loss": 0.280454158782959, "acc": 0.8910534977912903, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.24354133009910583, "acc": 0.905119776725769, "length": 3.9993674755096436, "mode": "train", "epoch": 28}
{"loss": 0.2840828597545624, "acc": 0.8966554403305054, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2424895167350769, "acc": 0.9070293307304382, "length": 3.999875783920288, "mode": "train", "epoch": 29}
{"loss": 0.2767951488494873, "acc": 0.8969898819923401, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.2395581305027008, "acc": 0.9082617163658142, "length": 3.998943328857422, "mode": "train", "epoch": 30}
{"loss": 0.26888081431388855, "acc": 0.8964882493019104, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.2373427152633667, "acc": 0.9072264432907104, "length": 3.9999747276306152, "mode": "train", "epoch": 31}
{"loss": 0.2717288136482239, "acc": 0.8985784649848938, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.23297937214374542, "acc": 0.9092627763748169, "length": 3.999967575073242, "mode": "train", "epoch": 32}
{"loss": 0.2588442265987396, "acc": 0.903261125087738, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.2293815314769745, "acc": 0.9118859171867371, "length": 3.9999918937683105, "mode": "train", "epoch": 33}
{"loss": 0.2587258517742157, "acc": 0.9015885591506958, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.22796085476875305, "acc": 0.9108197093009949, "length": 3.999387741088867, "mode": "train", "epoch": 34}
{"loss": 0.26572325825691223, "acc": 0.90167236328125, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.23759908974170685, "acc": 0.9048571586608887, "length": 3.998676300048828, "mode": "train", "epoch": 35}
{"loss": 0.28971394896507263, "acc": 0.8938964009284973, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.2345724254846573, "acc": 0.9090603590011597, "length": 3.9994986057281494, "mode": "train", "epoch": 36}
{"loss": 0.2708667814731598, "acc": 0.8957358002662659, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.2243257761001587, "acc": 0.9134742617607117, "length": 3.999842882156372, "mode": "train", "epoch": 37}
{"loss": 0.26721155643463135, "acc": 0.9056020975112915, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.21686896681785583, "acc": 0.9160378575325012, "length": 3.9995782375335693, "mode": "train", "epoch": 38}
{"loss": 0.25183144211769104, "acc": 0.9069397449493408, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.21848027408123016, "acc": 0.9145623445510864, "length": 3.9999117851257324, "mode": "train", "epoch": 39}
{"loss": 0.2733348309993744, "acc": 0.9010871052742004, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.21761371195316315, "acc": 0.9169620275497437, "length": 3.999877691268921, "mode": "train", "epoch": 40}
{"loss": 0.26342883706092834, "acc": 0.9066053628921509, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.21674537658691406, "acc": 0.9176287055015564, "length": 3.9996166229248047, "mode": "train", "epoch": 41}
{"loss": 0.2642460763454437, "acc": 0.9056022763252258, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.21033158898353577, "acc": 0.9201922416687012, "length": 3.999452590942383, "mode": "train", "epoch": 42}
{"loss": 0.2360079288482666, "acc": 0.9128763675689697, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2062940001487732, "acc": 0.9212605357170105, "length": 3.9999852180480957, "mode": "train", "epoch": 43}
{"loss": 0.24329732358455658, "acc": 0.9099498987197876, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.20054732263088226, "acc": 0.9250554442405701, "length": 3.9999866485595703, "mode": "train", "epoch": 44}
{"loss": 0.2508569657802582, "acc": 0.9119563698768616, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.19588546454906464, "acc": 0.9265332818031311, "length": 3.999948263168335, "mode": "train", "epoch": 45}
{"loss": 0.24424444139003754, "acc": 0.9091137647628784, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.20449283719062805, "acc": 0.9222611784934998, "length": 3.999892473220825, "mode": "train", "epoch": 46}
{"loss": 0.2554662823677063, "acc": 0.9045150876045227, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.20319844782352448, "acc": 0.9239651560783386, "length": 3.9999070167541504, "mode": "train", "epoch": 47}
{"loss": 0.27407220005989075, "acc": 0.9085283875465393, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.198625385761261, "acc": 0.9269508719444275, "length": 3.9998154640197754, "mode": "train", "epoch": 48}
{"loss": 0.2587643563747406, "acc": 0.9040133357048035, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.19138243794441223, "acc": 0.9264966249465942, "length": 3.9997589588165283, "mode": "train", "epoch": 49}
{"loss": 0.2497645616531372, "acc": 0.9105350971221924, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.19354797899723053, "acc": 0.9259679913520813, "length": 3.999608278274536, "mode": "train", "epoch": 50}
{"loss": 0.24353599548339844, "acc": 0.9098661541938782, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.19215859472751617, "acc": 0.9266329407691956, "length": 3.9997286796569824, "mode": "train", "epoch": 51}
{"loss": 0.2627575695514679, "acc": 0.9053511023521423, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.19968567788600922, "acc": 0.9234030842781067, "length": 3.9996964931488037, "mode": "train", "epoch": 52}
{"loss": 0.23439092934131622, "acc": 0.9110367894172668, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.1907608062028885, "acc": 0.9273843169212341, "length": 3.999084711074829, "mode": "train", "epoch": 53}
{"loss": 0.22667458653450012, "acc": 0.9163044691085815, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.1846153885126114, "acc": 0.9318903088569641, "length": 3.9999678134918213, "mode": "train", "epoch": 54}
{"loss": 0.22860954701900482, "acc": 0.9163042902946472, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.18602780997753143, "acc": 0.929129421710968, "length": 3.9999945163726807, "mode": "train", "epoch": 55}
{"loss": 0.23638026416301727, "acc": 0.9156352281570435, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.18120689690113068, "acc": 0.9332307577133179, "length": 3.9999947547912598, "mode": "train", "epoch": 56}
{"loss": 0.23742417991161346, "acc": 0.9208194613456726, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.18234683573246002, "acc": 0.9334263205528259, "length": 3.999995708465576, "mode": "train", "epoch": 57}
{"loss": 0.22531543672084808, "acc": 0.9204012751579285, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.17859476804733276, "acc": 0.9337610602378845, "length": 3.999990463256836, "mode": "train", "epoch": 58}
{"loss": 0.21700909733772278, "acc": 0.9204847812652588, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.175789013504982, "acc": 0.9359073638916016, "length": 3.9999032020568848, "mode": "train", "epoch": 59}
{"loss": 0.21485371887683868, "acc": 0.9239132404327393, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.1705234944820404, "acc": 0.9377506971359253, "length": 3.999976873397827, "mode": "train", "epoch": 60}
{"loss": 0.22746753692626953, "acc": 0.9144648313522339, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.1718098521232605, "acc": 0.9371925592422485, "length": 3.999971628189087, "mode": "train", "epoch": 61}
{"loss": 0.2247057408094406, "acc": 0.9200668334960938, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.17270605266094208, "acc": 0.9369696974754333, "length": 3.9999961853027344, "mode": "train", "epoch": 62}
{"loss": 0.2135632485151291, "acc": 0.9229095578193665, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.16735349595546722, "acc": 0.9398427605628967, "length": 3.9999754428863525, "mode": "train", "epoch": 63}
{"loss": 0.2251480668783188, "acc": 0.9219062328338623, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.16412785649299622, "acc": 0.9402313232421875, "length": 3.9999406337738037, "mode": "train", "epoch": 64}
{"loss": 0.21624480187892914, "acc": 0.9250001311302185, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.16819746792316437, "acc": 0.938364565372467, "length": 3.999990463256836, "mode": "train", "epoch": 65}
{"loss": 0.23301725089550018, "acc": 0.9193980693817139, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.17446449398994446, "acc": 0.9380854964256287, "length": 3.9999840259552, "mode": "train", "epoch": 66}
{"loss": 0.23086147010326385, "acc": 0.9199832081794739, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.16789038479328156, "acc": 0.9405288696289062, "length": 3.999786615371704, "mode": "train", "epoch": 67}
{"loss": 0.21384775638580322, "acc": 0.9272574186325073, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.16556625068187714, "acc": 0.9401697516441345, "length": 3.999465227127075, "mode": "train", "epoch": 68}
{"loss": 0.2349916696548462, "acc": 0.921655535697937, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.16275054216384888, "acc": 0.9417130351066589, "length": 3.99999737739563, "mode": "train", "epoch": 69}
{"loss": 0.21837487816810608, "acc": 0.9295151233673096, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.1568727195262909, "acc": 0.9449216723442078, "length": 3.9999961853027344, "mode": "train", "epoch": 70}
{"loss": 0.222959503531456, "acc": 0.924163818359375, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.16356927156448364, "acc": 0.9414339065551758, "length": 3.999993085861206, "mode": "train", "epoch": 71}
{"loss": 0.20673854649066925, "acc": 0.9270066618919373, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.15829706192016602, "acc": 0.944271981716156, "length": 3.9998490810394287, "mode": "train", "epoch": 72}
{"loss": 0.21238605678081512, "acc": 0.9316890835762024, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.15339027345180511, "acc": 0.9467688798904419, "length": 3.9992451667785645, "mode": "train", "epoch": 73}
{"loss": 0.1976650357246399, "acc": 0.9329431056976318, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.15776227414608002, "acc": 0.9444352388381958, "length": 3.9993896484375, "mode": "train", "epoch": 74}
{"loss": 0.20887203514575958, "acc": 0.9305182099342346, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.15143339335918427, "acc": 0.9475719332695007, "length": 3.999969720840454, "mode": "train", "epoch": 75}
{"loss": 0.20063434541225433, "acc": 0.9376252293586731, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.1473405808210373, "acc": 0.9489287734031677, "length": 3.9994585514068604, "mode": "train", "epoch": 76}
{"loss": 0.20443756878376007, "acc": 0.9349498152732849, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.1478852480649948, "acc": 0.9474557042121887, "length": 3.9997313022613525, "mode": "train", "epoch": 77}
{"loss": 0.2070094347000122, "acc": 0.9319397807121277, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.15236103534698486, "acc": 0.9458975195884705, "length": 3.9999632835388184, "mode": "train", "epoch": 78}
{"loss": 0.20873136818408966, "acc": 0.9336954355239868, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.16030751168727875, "acc": 0.9428847432136536, "length": 3.9999918937683105, "mode": "train", "epoch": 79}
{"loss": 0.26375797390937805, "acc": 0.9190635085105896, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.15542851388454437, "acc": 0.9451410174369812, "length": 3.9998812675476074, "mode": "train", "epoch": 80}
{"loss": 0.20576779544353485, "acc": 0.9297658801078796, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.14432315528392792, "acc": 0.9494665861129761, "length": 3.999901056289673, "mode": "train", "epoch": 81}
{"loss": 0.20359273254871368, "acc": 0.9332776665687561, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.14022581279277802, "acc": 0.9499433636665344, "length": 3.999979257583618, "mode": "train", "epoch": 82}
{"loss": 0.1877145618200302, "acc": 0.9368731379508972, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.13709525763988495, "acc": 0.9506973624229431, "length": 3.999994993209839, "mode": "train", "epoch": 83}
{"loss": 0.1920192688703537, "acc": 0.9362876415252686, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.13798484206199646, "acc": 0.9515066146850586, "length": 3.999998092651367, "mode": "train", "epoch": 84}
{"loss": 0.19902358949184418, "acc": 0.9351170659065247, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.13569073379039764, "acc": 0.9506970643997192, "length": 3.9999771118164062, "mode": "train", "epoch": 85}
{"loss": 0.1902148425579071, "acc": 0.9400501251220703, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.13642996549606323, "acc": 0.9527729153633118, "length": 3.9989168643951416, "mode": "train", "epoch": 86}
{"loss": 0.1898607313632965, "acc": 0.9391303658485413, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.14557181298732758, "acc": 0.9490789175033569, "length": 3.999986410140991, "mode": "train", "epoch": 87}
{"loss": 0.20969124138355255, "acc": 0.928762674331665, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.14849145710468292, "acc": 0.9464285969734192, "length": 3.999999523162842, "mode": "train", "epoch": 88}
{"loss": 0.19061259925365448, "acc": 0.9386288523674011, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.13720116019248962, "acc": 0.9519248604774475, "length": 3.9999966621398926, "mode": "train", "epoch": 89}
{"loss": 0.20661501586437225, "acc": 0.9335284233093262, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.13749364018440247, "acc": 0.9504740834236145, "length": 3.999990224838257, "mode": "train", "epoch": 90}
{"loss": 0.19933167099952698, "acc": 0.9391304850578308, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.13500390946865082, "acc": 0.9550963640213013, "length": 3.9998345375061035, "mode": "train", "epoch": 91}
{"loss": 0.20688214898109436, "acc": 0.9341138601303101, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.13201117515563965, "acc": 0.9522590637207031, "length": 3.9999759197235107, "mode": "train", "epoch": 92}
{"loss": 0.20520372688770294, "acc": 0.9386287331581116, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.132583886384964, "acc": 0.9542967677116394, "length": 3.9999961853027344, "mode": "train", "epoch": 93}
{"loss": 0.17949619889259338, "acc": 0.946237325668335, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.12503105401992798, "acc": 0.9575331807136536, "length": 3.9999923706054688, "mode": "train", "epoch": 94}
{"loss": 0.19143374264240265, "acc": 0.9407191276550293, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.12485365569591522, "acc": 0.9576169848442078, "length": 3.9999959468841553, "mode": "train", "epoch": 95}
{"loss": 0.1842566579580307, "acc": 0.9430602192878723, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.12672176957130432, "acc": 0.9577563405036926, "length": 3.999994993209839, "mode": "train", "epoch": 96}
{"loss": 0.20350341498851776, "acc": 0.9407190680503845, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.12759056687355042, "acc": 0.95750492811203, "length": 3.9999897480010986, "mode": "train", "epoch": 97}
{"loss": 0.18709111213684082, "acc": 0.9438963532447815, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.11837784200906754, "acc": 0.960600733757019, "length": 3.9999423027038574, "mode": "train", "epoch": 98}
{"loss": 0.19250552356243134, "acc": 0.9428929090499878, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.11629828065633774, "acc": 0.9617681503295898, "length": 3.9997975826263428, "mode": "train", "epoch": 99}
{"loss": 0.19413107633590698, "acc": 0.9412205815315247, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.11425793170928955, "acc": 0.9614396691322327, "length": 3.999999523162842, "mode": "train", "epoch": 100}
{"loss": 0.18787410855293274, "acc": 0.9422240257263184, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.808524489402771
